components/Modal/modal.html

```diff
@@ -0,0 +1,98 @@
+                </p>
+            </section>
+
+            <section>
+                <h2 class="section-heading">Basic Modal</h2>
+                <p class="section-para">
+                    This is a simple modal displaying heading and sub-heading and ask among two choices from a user.
+                </p>
+                <div class="modal-container">
+                    <h3 class="modal-heading">Modal Heading</h3>
+                    <p class="modal-subheading">Modal subheading displaying certain piece of information.</p>
+                    <div class="modal-btns">
+                        <button class="btn btn-solid-primary">Action 1</button>
+                        <button class="btn btn-outline-primary">Action 2</button>
+                    </div>
+                </div>
+                <div class="iframe-wrapper iframe-md">
+                    <iframe class="iframe-code"
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522modal-container%2522%253E%250A%2520%2520%2520%2520%253Ch3%2520class%253D%2522modal-heading%2522%253EModal%2520Heading%253C%252Fh3%253E%250A%2520%2520%2520%2520%253Cp%2520class%253D%2522modal-subheading%2522%253EModal%2520subheading%2520displaying%2520certain%2520piece%2520of%2520information.%253C%252Fp%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522modal-btns%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-solid-primary%2522%253EAction%25201%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-outline-primary%2522%253EAction%25202%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </section>
+
+        </section>
+
+    </main>
+
+    <footer class="footer">
+
+        <div class="footer-container">
+            <p class="footer-title">Made by Atul Pant</p>
+            <div class="social-media">
+                <a href="https://github.com/AtulPant2704" target="_blank"><i class="fab fa-github"></i></a>
+                <a href="https://linkedin.com/in/atulpant27" target="_blank"><i class="fab fa-linkedin"></i></a>
+                <a href="https://twitter.com/AtulPant2704" target="_blank"><i class="fab fa-twitter"></i></a>
+            </div>
+            <p class="footer-end">© 2022 CheckMate Designs</p>
+        </div>
+
+    </footer>
+
+    <script src="../../js/header.js"></script>
+    
+</body>
+
+</html>
```

Comment:
You could add a blank line at the end to avoid such warnings from Github

Reply:
I would have done that, but for that then I would have to do that in my each and every file to have same indentation which would require a lot of time and I don't have any as of now. Thanks for the suggestion, will take care of it from next project onwards.